Number swapping

sanjaypradeep opened the pull request:
Swapping of numbers is very easy in Python when compare to other languages like Java, etc

Commit:
Testing

Testing

Commit:
Swapping of numbers is very easy in Python when compare to other languages like Java, etc

hmangukia commented:
Instead of "raw_input()", use 'input()' and type a message for the user to enter the first value.

sanjaypradeep committed:
Addressed comments. 

raw_input() is replaced with input() with proper messages asking input to user.
